### PR TITLE
Tidy up the minikube platform setup documentation

### PR DESCRIPTION
**Pull request by sdake**
One minor issue was reported - however this is a bit of a rework of
the minikube documentation.  Additionally included is the LoadBalancer
feature `minikube tunnel`.

### CODE REVIEW

**Comment by sdake** [comment]
Linting error is unrelated

```
- ./public/zh/blog/2018/delayering-istio/index.html
  *  External link https://www.haproxy.org/download/1.8/doc/proxy-protocol.txt failed: got a time out (response code 0)
htmlproofer 3.9.2 | Error:  HTML-Proofer found 1 failure!
LINTING FAILED
Exited with code 1
```

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 1]
```diff
@@ -8,22 +8,44 @@ aliases:
...
    [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.28.1 or
    later**.
 
-1. Select a
-   [VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
-   and substitute `your_vm_driver_choice` below with the installed virtual
-   machine (VM) driver. To install Istio control plane components and addons,
-   as well as other applications,
-   we recommend starting Minikube with 8192 `MB` of memory and 4 `CPUs`:
+1. Select a [Minikube VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: Do we really want to dwell on this vm driver stuff here? It's just part of installing minikube. I myself use virtualbox which is the default, so I don't even need to set the `--vm-driver` and this whole part will just be noise.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 2]
```diff
@@ -8,22 +8,44 @@ aliases:
...
+   and install it using the driver installation techniques.  Finally export the
+   driver to the environment.  An example using kvm2 driver is:
 
-    On Kubernetes **1.13**:
+    {{< text bash >}}
```
Comment (ericvn) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: Another option is to run `minikube config set vm-driver kvm2` and not have to add to your profile.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 3]
```diff
@@ -8,22 +8,44 @@ aliases:
...
+    $ export MINIKUBE_VM_DIVER=kvm2
+    {{< /text >}}
+
+    {{< idea >}}
+    You may additionally choose to add the `MINIKUBE_VM_DRIVER` export to your `.profile` shell.  By
```
Comment (ericvn) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: Another option is to run minikube config set vm-driver kvm2 and not have to add to your profile.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 4]
```diff
@@ -8,22 +8,44 @@ aliases:
...
+1. (Optional) If you want a Load balancer in Minikube, you can use the
+   [Minikube Tunnel](https://github.com/kubernetes/minikube/blob/master/docs/tunnel.md).
+
+    {{< text bash >}}
+    $ minikube tunnel
+    {{< /text >}}
+
+    {{< tip >}}
+    Running the minikube tunnel feature will block your shell and output diagonstic information.  Be
+    prepared to run this optional command in a different shell.
```
Comment (ericvn) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: c/shell/terminal window/ ?

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 3 above)
Comment (frankbu) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: Yes, this sounds good to me. There should be an optional step (or maybe just an info box) that says to `minikube config` your vm-driver, if necessary. Then the following `minikube start` command doesn't even need to include the `--vm-driver` arg.

Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: sounds better than what I got.  I will make it so. 👍

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 1 above)
Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: I don't think we want to dwell on it, but we do want to make sure people don't come back and say "what you said doesn't work".  For example, I use VMWare's mac hypervisor...

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 5]
```diff
@@ -8,22 +8,40 @@ aliases:
...
 
-Follow these instructions to prepare Minikube for Istio.
+Follow these instructions to prepare Minikube for Istio installation with sufficient
+resources to run Istio and some basic applications.
 
 1. To run Istio locally, install the latest version of
-   [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.28.1 or
-   later**.
+   [Minikube](https://kubernetes.io/docs/setup/minikube/), version **0.33.1 or
+   later** and a
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]:
```suggestion
   later**, and a
```

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 6]
```diff
@@ -8,22 +8,40 @@ aliases:
...
+   [Minikube Hypervisor Driver](https://kubernetes.io/docs/tasks/tools/install-minikube/#install-a-hypervisor).
 
-1. Select a
-   [VM driver](https://kubernetes.io/docs/setup/minikube/#quickstart)
-   and substitute `your_vm_driver_choice` below with the installed virtual
-   machine (VM) driver. To install Istio control plane components and addons,
-   as well as other applications,
-   we recommend starting Minikube with 8192 `MB` of memory and 4 `CPUs`:
+    {{< tip >}}
+    Set your Minikube Hypervisor driver.  For example if you installed the KVM hypervisor, set the vm-driver
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]:
```suggestion
    Set your Minikube Hypervisor driver. For example, if you installed the KVM hypervisor, set the vm-driver
```

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 7]
```diff
@@ -8,22 +8,40 @@ aliases:
...
+    within the Minikube configuration:
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]:
```suggestion
    within the Minikube configuration using the following command:
```

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 8]
```diff
@@ -8,22 +8,40 @@ aliases:
...
 
-    On Kubernetes **1.13**:
+    {{< text bash >}}
+    $ minikube config set vm-driver kvm2
+    {{< /text >}}
+
+    {{< /tip >}}
+
+1. Start Minikube with 8192 `MB` of memory and 4 `CPUs`.  This example uses Kuberenetes
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]:
```suggestion
1. Start Minikube with 8192 `MB` of memory and 4 `CPUs`. The following example uses Kuberenetes
```

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 9]
```diff
@@ -8,22 +8,40 @@ aliases:
...
+   vesion 1.13.0, however, you may alter the version to Kubernetes versions of Istio that are
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]:
```suggestion
   version 1.13.0, however, you can change the version to any Kubernetes version
```

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md [hunk 10]
```diff
@@ -8,22 +8,40 @@ aliases:
...
+   supported:
```
Comment (frankbu) [inline content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]:
```suggestion
   supported by Istio:
```

**Comment by sdake** [comment]
> A few grammatical suggestions, but otherwise looks good.
> 
> Btw, I'm surprised that the typo (`vesion` on line 30) didn't produce a build error. Any idea @geeknoid ?

thanks @frankbu.  Note I am having serious keyboard problems on my laptop atm and plan to send it in for  repair.  until than, please take a close look at my doc PRs as they may contain typos or duplicate characters.  Mac laptops ftl.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 4 above)
Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: I shortened this to terminal.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 5 above)
Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: fixed.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 7 above)
Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: fixed.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 9 above)
Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: fixed.

File: content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md (hunk 10 above)
Comment (sdake) [reply content/docs/setup/kubernetes/prepare/platform-setup/minikube/index.md]: fixed.

**Pull request merged**